Library/Formula/midnight-commander.rb

```diff
@@ -28,6 +28,7 @@ def install
                           "--with-screen=slang",
                           "--enable-vfs-sftp"
     system "make", "install"
+    inreplace libexec/"mc/ext.d/text.sh", "man -P cat -l ", "man -P cat "
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!